### DISCUSSION TITLE
whl is not a supported wheel on this platform

### DISCUSSION THREAD

**Discussion by JeanLucCurilla** [thliebig/openEMS-Project#125] [Q&A Usage]
Hello,

I am trying to install openEMS on a Window 10 system and I got this message:

ERROR: CSXCAD-0.6.2-cp310-cp310-win_amd64.whl is not a supported wheel on this platform.

I think that I used a bad version of Python (3.12) and pip  (23.2.1) ! 

Therefore could you tell me what is the right environment for this installation?

My configuration:

Processor: Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz   2.40 GHz
Memory: 8 Go
OS System: Windows 10 Professional 22H2

Best regards!

**Comment by thliebig** [comment]
You are trying to install a python 3.10 wheel (cp310) with python 3.12, that will not work.
I think I currently only supply a wheel for 3.10 and 3.11. I would recommend you use one of that versions, e.g. using [winpyhon](https://winpython.github.io/).

**Reply by ahmedmibrahim** [reply]
I'm using 3.11.1 and still fails


PS C:\Users\XXX> python --version
Python 3.11.1
PS C:\Users\XXX> pip install CSXCAD-0.6.2-cp310-cp310-win_amd64.whl
WARNING: Requirement 'CSXCAD-0.6.2-cp310-cp310-win_amd64.whl' looks like a filename, but the file does not exist
WARNING: Ignoring invalid distribution ~latformio (C:\Users\XXX\.platformio\penv\Lib\site-packages)
ERROR: CSXCAD-0.6.2-cp310-cp310-win_amd64.whl is not a supported wheel on this platform.
PS C:\Users\XXX>

**Reply by thliebig** [reply]
You are trying to install the cp310 (aka 3.10) in a python 3.11...

**Comment by JeanLucCurilla** [comment]
Dear Thorsten Liebig,

Thank you very much for your answer. Now, with Python 3.11 it works without difficulties.

Best regards, Jean-Luc Curilla.

De : Thorsten Liebig ***@***.***>
Envoyé : dimanche 8 octobre 2023 20:11
À : thliebig/openEMS-Project ***@***.***>
Cc : CURILLA Jean-Luc ***@***.***>; Author ***@***.***>
Objet : Re: [thliebig/openEMS-Project] whl is not a supported wheel on this platform (Discussion #125)


You are trying to install a python 3.10 wheel (cp310) with python 3.12, that will not work.
I think I currently only supply a wheel for 3.10 and 3.11. I would recommend you use one of that versions, e.g. using winpyhon<https://winpython.github.io/>.

-
